src/components/home/MoviePlay.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import PlusIcon from "#/icons/home/plus.svg";
+import PlayIcon from "#/icons/home/play.svg";
+import InfoIcon from "#/icons/home/info.svg";
+import { GetMoviesResponse } from "@/types/home";
+
+interface MoviePlayProps {
+  movieData: GetMoviesResponse;
+}
+const MoviePlay = ({ movieData }: MoviePlayProps) => {
+  const posterBaseUrl = process.env.NEXT_PUBLIC_POSTER_BASE_URL;
+  const [randomIndex, setRandomIndex] = useState<number>(0);
+
+  useEffect(() => {
+    const index = Math.floor(Math.random() * movieData.results.length);
+    setRandomIndex(index);
+  }, [movieData]);
```

Comment:
해당 컴포넌트에서 랜덤 index를 세팅하는 상세 로직 자체는 그렇게 중요한 부분이 아니기 떄문에 커스텀 훅에서 해당 로직을 실행하고 index만 넘겨줘도 좋은 코드가 될 것 같아요 ㅎㅎ

useRandomIndex 훅 생성
```
export default function UseRandomIndex(movieData) {
   const [randomIndex, setRandomIndex] = useState<number>(0);

   useEffect(() => {
     const index = Math.floor(Math.random() * movieData.results.length);
     setRandomIndex(index);
  }, [movieData]);

   return randomIndex;
}

```

```suggestion
  const randomIndex = useRandomIndex(movieData);
```